book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(person, date)
+    Rental.new(person, date, self)
```

Comment:
- [ ] To avoid errors, we need to make these arguments match how they are defined in the rental constructor
  
  ```
  Rental.new(date, self, person)
  ```